catalogue/webapp/pages/concept.tsx

```diff
@@ -86,9 +87,17 @@ const ConceptWorksHeader = styled(Space).attrs({
     theme.color(hasWorksTabs ? 'warmNeutral.300' : 'white')};};
 `;
 
-const SeeMoreButton = ({ text, link }: { text: string; link: LinkProps }) => (
+const SeeMoreButton = ({
+  text,
+  link,
+  totalResults,
+}: {
+  text: string;
+  link: LinkProps;
+  totalResults: number;
+}) => (
   <MoreLink
-    name={text}
+    name={text + ` (${formatNumber(totalResults)})`}
```

Comment:
Don't mix interpolation styles in the same line:

```suggestion
    name={`${text} (${formatNumber(totalResults)})`}
```